reference/7/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -155,53 +157,58 @@ PowerShell supports the following type accelerators:
 > The following type accelerators were added in PowerShell 6.2: `[short]`,
 > `[ushort]`, `[uint]`, `[ulong]`.
 
-### Working with other numeric types
-
-To work with any other numeric types you must use type accelerators, which is
-not without some problems. For example, high integer values are always parsed
-as double before being cast to any other type.
-
-```
-PS> [bigint]111111111111111111111111111111111111111111111111111111
-111111111111111100905595216014112456735339620444667904
-```
-
-The value is parsed as a double first, losing precision in the higher ranges.
-To avoid this problem, enter values as strings and then convert them:
-
-```
-PS> [bigint]'111111111111111111111111111111111111111111111111111111'
-111111111111111111111111111111111111111111111111111111
-```
-
```

Comment:
Rather than remove this whole section, I might be more inclined to simply mention use of the `n` suffix as a much simpler alternative to the string->cast method. I think there's still value in having it explicitly mentioned like this for users who aren't yet aware that this rounding occurs.

It might also be worth a mention that some values between `[ulong]::MaxValue` and `[decimal]::MaxValue` may also be denoted as `Decimal` by suffix to maintain accuracy. These values are usually parsed as `Double` due to the `real`-parsing mode not considering `decimal` without specifically using the suffix.

Reply:
Got it.